dataService/build/pluginAPI64.x

```diff
@@ -0,0 +1,143 @@
+ IMPORT CODE64,'zssServer64','makeHttpDataService'                             
```

Comment:
TODO for a later day... we should revisit ZSS issue 30 to come up with a well-tailored list of "functions we want people to use" and "functions we dont" and even make nicely named aliases for things we want people to use, and publish the whole thing so that instead of each repo having their own special and easy-to-become-outdated side file, they could rely upon a central one...

Reply:
Don't worry about fixing this, just a mental note.